udata_search_service/presentation/api.py

```diff
@@ -49,6 +49,7 @@ class DatasetArgs(BaseModel):
     temporal_coverage: Optional[str] = None
     featured: Optional[str] = None
     schema_: Optional[str] = Field(None, alias="schema")
+    topic: Optional[str] = None
```

Comment:
I think we should add the filter transformation from topic -> topics in `format_filters`. (same as tags)